addons/bestja_page_fixtures_ucw/templates/pages/become_volunteer.xml

```diff
@@ -155,9 +155,10 @@
                                         <p>Jeżeli będziesz zainteresowany
                                             konkretnym ogłoszeniem skontaktujemy
                                             Cię z koordynatorami wybranej organizacji.</p>
-                                        <form action="/web/signup">
-                                            <button class="btn btn-lg mt32 mb32 button_orange" type="submit">DOŁĄCZ TERAZ</button>
+                                        <form t-if="user_id.partner_id.name == 'Public user'" action="/web/signup">
```

Comment:
It's not a good practice to test this by name. Name can changed or can be translated. Here is how Odoo's built-in website module does this:

    t-if="website.user_id != user_id"

(`website.user_id` is the "public" user configured for the particular website)